ansible/roles/mixing/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+# Mixing
+
+- name: Get wallet funding address
+  ansible.builtin.command:
+    cmd: dash-cli -rpcwallet=coinjoin-wallet-{{ item | string }} getnewaddress
+  register: getnewaddress_result
+  loop: "{{ range(0, coinjoin_wallet_count | int, 1) }}"
+
+- name: Debug
+  debug:
+    var: getnewaddress_result
+
+- name: Prepare list of CoinJoin funding targets
+  ansible.builtin.set_fact:
+    coinjoin_funding_addresses: "{{ (coinjoin_funding_addresses | default([])) + [item.stdout] }}"
+  with_items: "{{ getnewaddress_result.results }}"
+
+- name: Fund 10 coins for mixing
+  ansible.builtin.include_tasks: ./roles/mn_fund_collateral/tasks/fund_collateral.yml
+  vars:
+    amount: 10
+    payment_targets: '{{ coinjoin_funding_addresses }}'
+
+- name: Get wallet balance
+  ansible.builtin.command:
+    cmd: dash-cli -rpcwallet=coinjoin-wallet-{{ item | string }} getbalance
+  register: getbalance_result
+  loop: "{{ range(0, coinjoin_wallet_count | int, 1) }}"
+
+- name: Set CoinJoin amount
+  ansible.builtin.command:
+    cmd: dash-cli -rpcwallet=coinjoin-wallet-{{ item | string }} setcoinjoinamount 10
+  loop: "{{ range(0, coinjoin_wallet_count | int, 1) }}"
+
+- name: Set CoinJoin rounds
+  ansible.builtin.command:
+    cmd: dash-cli -rpcwallet=coinjoin-wallet-{{ item | string }} setcoinjoinrounds 16
```

Comment:
Let's make the rounds a variable to be set in network config

Reply:
@ktechmidas fixed in https://github.com/dashpay/dash-network-deploy/pull/528/commits/508d6866de003408600933fd3a78133a51e38c0e